### ISSUE TITLE
[CPU] operator_benchmark: backward-pass (add, + batchnorm on aarch64) ~200-500x slower since ~Nov 2025 (x86 + aarch64; possible measurement artifact)

### ISSUE THREAD

**Issue by frgossen** [pytorch/pytorch#189282]
Possible / low-confidence. `add_M64_N64_K*_cpu_bwd*_BACKWARD`, 9 cases, ~200-400x slower vs the prior baseline, stable across runs.

Flagged as low-confidence: a real 400x regression in `add` backward would be catastrophic and caught in training, so this looks more like a backward-pass measurement artifact than a genuine op regression -- but filing defensively for someone who knows the benchmark harness / autograd path.

```
add_M64_N64_K128_cpu_bwd2_BACKWARD   old 135.8 -> 57315 / 57528  (~424x)
add_M64_N64_K64_cpu_bwd2_BACKWARD    old 135.2 -> 30762 / 30904  (~229x)
```

## Context
Surfaced while rebaselining the operator_benchmark CPU perf baselines (PR #189079, tracking #189041). The previous baseline was last updated on 2025-11-10 (#167099), so the regression window is roughly 2025-11-10 .. 2026-07. Job: `x86-opbenchmark-test` (eager, float32, CPU). Values below are `old baseline -> run 2026-07-05 / run 2026-07-06` (microseconds); the two recent runs agree with each other (stable, not run-to-run noise).

Filed defensively by pt2-oncall triage -- we have not investigated the code path. Passing to an owner who knows this area to confirm whether it is a real regression.

## Reproduce
Run the op-benchmark suite and compare against the pre-#189079 baseline:
```
python -m pt.<...> --tag-filter short   # via .ci/pytorch/test.sh test_operator_benchmark
```

cc @ezyang @albanD @gqchen @nikitaved @soulitzer @Varal7 @bobrenjc93 @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @aditew01

**Comment by frgossen** [comment]
Also reproduces on **aarch64** (`aarch64-opbenchmark-test`), stable across the 2026-07-05 / 2026-07-06 runs. On aarch64 the same `add` backward cases regress, **and batchnorm backward joins the pattern** (batchnorm is skip-listed / not flagged on x86):

```
add_M64_N64_K128_cpu_bwd2_BACKWARD                       old 250.8 -> 74653 / 76049    (~303x)
batchnorm_N3136_C256_cpu_trainingTrue_cudnnFalse_bwdall  old 484.4 -> 224658 / 229262  (~473x)
batchnorm_M1_N256_K3136_cpu_trainingTrue_cudnnFalse_bwd  old 335.4 -> 148016 / 150129  (~448x)
```
Same low-confidence framing: a real 200-500x backward regression is implausible, so this points at a backward-pass measurement artifact -- but it is shared across arches and includes batchnorm backward on aarch64.

**Comment by albanD** [comment]
cc @jathu operator benchmark is in a sad state and it is unclear if it is an infra problem or not? Also to understand if we should disable these benchmarks if they're expensive.

Also from Nikita, we should track all of these as a single umbrela issue.
